test/valgrind/valgrindTestSuite.sh

```diff
@@ -185,8 +185,14 @@ do
   elif [ "$1" == "-fromIx" ];   then  shift; fromIx=$1;
   elif [ "$1" == "-ixList" ];   then  shift; ixList=$1;
   else
-    echo $0: bad parameter/option: "'"${1}"'";
-    usage 1
+    if [ "$file" == "" ]
+    then
+      file=$1
+      TEST_FILTER=$file
+    else
+      echo $0: bad parameter/option: "'"${1}"'";
+      usage 1
+    fi
```

Comment:
Stetical change / minor improvement? Or something reallly related with the fixing of memory accesses errores?

Reply:
See the PR description:
_Made valgrind test-suite accept a file as parameter (like the functional test harness), not having to use the -filter option anymore for this common case (the option still works)_

Got sick and tired of calling valgrindTestSuite.sh with `-filter <test case>`, so I made the test suite more similar to testHarness.sh that accepts a parameter that is a test case file, without option selector. Just *one* though, that's why an error is signaled if there is more than one

Reply:
Usage message need to be changed due to the new "CLI API" ?

Reply:
Ah, yeah, good idea (even though I think only I use the script :-))

Reply:
Fixed in 8cb2104